essential-knowledge/cp-permissions.rst

```diff
@@ -0,0 +1,90 @@
+.. _knowledge-cp-permissions:
+
+Control panel permissions reference
+====================================
+
+The following table shows the different permission options for the different
+types of users with respect to addons, boilerplates and applications.
+
+
+Addons
+----------
+
++---------------+-------------+--------------+---------------+---------------+-----------------+-----------------+
```

Comment:
There is a `..list-table` directive which may be useful for *editing*: https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table

Reply:
I have used both styles, the List-table and the Grid-table. I would go for the Grid table, even though tedious to write, it has better features. Let me know your thoughts.